### DISCUSSION TITLE
The item "Remote Desktop Connection" in the form of a menu.

### DISCUSSION THREAD

**Discussion by paolosezart** [Open-Shell/Open-Shell-Menu#879] [Q&A] [answered]
How to make the "Remote Desctop Connection" item in the form of a menu with a drop-down list of recent connections? I recently reinstalled Windows 7. I remember in the old version there was a drop-down menu with the ability to pin some connections. For now, it's just a link. I use this very often and I wanted to do it as before, because it is much more convenient that way.
I apologize for my stupidity and thank you in advance for the answer.

**Comment by Gaurav-Original-ClassicShellTester** [comment, marked as answer]
You already asked it here: https://github.com/Open-Shell/Open-Shell-Menu/issues/865 Anyway I answered it there and will answer it here again: http://www.classicshell.net/forum/viewtopic.php?f=7&t=8529 Follow all the steps in all the posts in that thread. You have to enable jumplists in the operating system settings.

**Comment by paolosezart** [comment]
I am very sorry. I didn't see your previous post.
This checkbox is inactive for me:
![Скриншот 15-10-2021 235023](https://user-images.githubusercontent.com/38177890/137538450-c1abee4e-2a4f-4d8f-a11f-cb2d609b8c57.jpg)
I have Windows 7 Professional installed.

**Reply by Gaurav-Original-ClassicShellTester** [reply]
Maybe some Group Policies are enabled on the system. https://answers.microsoft.com/en-us/windows/forum/all/jumplists-disabled-option-to-store-and-display/1c05dc81-4983-48e3-a606-6effd64e611f?auth=1

https://www.sevenforums.com/tutorials/32666-recent-programs-enable-disable.html

Maybe one of the programs you use to tweak Windows set the policies.

**Reply by paolosezart** [reply]
I did as you said:
![Скриншот 16-10-2021 164537](https://user-images.githubusercontent.com/38177890/137586215-e258530c-1d39-45e5-ac27-7e8dd6af49f1.jpg)

But still the menu does not appear, only the link:

![Скриншот 16-10-2021 161746](https://user-images.githubusercontent.com/38177890/137586274-c400b2a2-2ca4-42d2-b0bb-6850787a8e0e.jpg)

At the same time, Internet Explorer shows a menu.
This is how the RDP client shows itself, as you can see, there is a connection history:
![Скриншот 16-10-2021 161920](https://user-images.githubusercontent.com/38177890/137586456-feef5e0c-a851-4f7b-8acc-4a3c03748383.jpg)

How can I configure that there was a normal menu?

**Reply by Gaurav-Original-ClassicShellTester** [reply]
If the Taskbar shows jumplists, the shortcut for the Start Menu will also show them. Does the taskbar show it? If you just turned them on, only the items added to the jumplist after you turned them on will show in the menu. Only when they appear in Recent, you can pin them.

**Reply by paolosezart** [reply]
> If the Taskbar shows jumplists, the shortcut for the Start Menu will also show them. Does the taskbar show it?

I do not quite understand you.
Could you show it in the screenshot?
Thank you in advance!

**Reply by Gaurav-Original-ClassicShellTester** [reply]
When you start Remote Desktop, and then right click its button on the taskbar to show its jumplist, do you get them there in that list or nope? That jumplist data is collected, maintained, updated and handled entirely by Windows, depending on OS settings. The Start menu only shows that same jumplist as the taskbar and can modify/update it.

If you want to pin certain thingies there, just find the appropriate *.rdp files for them and drag them over to the RDP icon on the taskbar to pin them.

**Reply by paolosezart** [reply]
> If you want to pin certain thingies there, just find the appropriate *.rdp files for them and drag them over to the RDP icon on the taskbar to pin them.

Thank you friend! This is the most ideal solution for my case.
I have been using windows for a long time, but I have not thought of such a simple solution.

**Comment by Ibuprophen** [comment]
Closing/Locking as this question was addressed in issue https://github.com/Open-Shell/Open-Shell-Menu/issues/865 .


~Ibuprophen
